install/kubernetes/helm/Istio/values.yaml

```diff
@@ -0,0 +1,13 @@
+Istio:
+   auth_enabled: false 
+   initializer_enabled: false
+   mixer_enabled: true
+   pilot_enabled: true
+   proxy_enabled: true
```

Comment:
this seems like an odd choice to provide the operator.  Could you expand on the rationale?

Reply:
No rational at all, it is to demonstrate that each component can be disabled/enabled which is in line at least I think with Istio deployment strategy.

Reply:
So this is also a required component. Essentially istio infrastructure today consists of proxies/sidecars, pilot. Then you can optionally add telemetry, tracing, policy, secure communication using mTLS